test/e2e/pageobjects/app-frontend.ts

```diff
@@ -19,6 +19,9 @@ export class AppFrontend {
 
     /** @see https://dev.altinn.studio/repos/ttd/payment-test */
     paymentTest: 'payment-test',
+
+    /** @see https://altinn.studio/repos/ttd/component-library.git */
+    componentLibrary: 'component-library',
```

Comment:
Remember to add a note about this app to the `README.md` as well. :raised_hands: